src/locale/locales/pt-BR/messages.po

```diff
@@ -6034,7 +6034,7 @@ msgstr "Repostado por você"
 
 #: src/view/com/notifications/FeedItem.tsx:180
 msgid "reposted your post"
-msgstr "repostou seu post"
+msgstr "repostaram sua postagem"
```

Comment:
Same as the comment about plural forms for liked posts.
Can I use something like this?
`msgstr "{repostCount, plural, one {repostou sua postagem} other {repostaram sua postagem}}"`